src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => resolve());
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error);
+  }, 3000);
+});
+
+function isSuccess() {
+  document.body.insertAdjacentHTML(
+    'beforeend',
+    `
+    <div class="message">
+      Promise was resolved!
+    </div>
+  `
+  );
+}
+
+function isRejected() {
```

Comment:
```suggestion
function handleError() {
```
Actually, these two functions are similar and can be combined into one function